packages/keystone/bin/commands/dev.js

```diff
@@ -1,46 +1,16 @@
 const keystone = require('@keystone-alpha/core');
-const endent = require('endent');
-const path = require('path');
-
-function getEntryFileFullPath(args, { exeName, _cwd }) {
-  const entryFile = args['--entry'] ? args['--entry'] : keystone.DEFAULT_ENTRY;
-  try {
-    return Promise.resolve(require.resolve(path.resolve(_cwd, entryFile)));
-  } catch (error) {
-    return Promise.reject(
-      new Error(endent`
-        --entry=${entryFile} was passed to ${exeName}, but '${entryFile}' couldn't be found in ${process.cwd()}.
-        Ensure you're running ${exeName} from within the root directory of the project.
-      `)
-    );
-  }
-}
-
-function executeDefaultServer(args, entryFile) {
-  const port = args['--port'] ? args['--port'] : keystone.DEFAULT_PORT;
-
-  return keystone
-    .prepare({ entryFile, port })
-    .then(async ({ server, keystone: keystoneApp }) => {
-      await keystoneApp.connect();
-
-      return server.start();
-    })
-    .then(() => {
-      console.log(`KeystoneJS ready on port ${port}`);
-    });
-}
+const { executeDefaultServer, getEntryFileFullPath } = require('../utils');
 
 module.exports = {
   // prettier-ignore
   spec: {
-    '--port':   Number,
-    '-p':       '--port',
-    '--entry':  String,
+    '--port':  Number,
+    '-p':      '--port',
+    '--entry': String,
   },
   help: ({ exeName }) => `
     Usage
-      $ ${exeName} dev --port=3000
+      $ ${exeName} dev <dist> --port=3000
```

Comment:
It doesn't look like `<dist>` here does anything (but it does in `start.js`)

```suggestion
      $ ${exeName} dev --port=3000
```